bin/test

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+rake parallel:test && rake parallel:features
```

Comment:
Just tried it. It still runs them in series, so identical either way. I'm not sure if there's any other advantage doing it one way or the other.

There are enough individual tests in each suite that all of your cores will be kept busy anyway, so I don't think we are going to gain much more by trying to get them to run in parallel.